composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.5.0",
```

Comment:
Actually, we could also bump the PHP version, and other Symfony dependencies.

Reply:
Please, let's do that. No need to still maintain such an old PHP version.

Reply:
Assuming this would be done in a separate MR, I propose #387  
Feel free to cherry-pick or whatever :)